libutils/ip_address.c

```diff
@@ -1235,31 +1235,34 @@ int IPAddressCompareLess(IPAddress *a, IPAddress *b)
      */
      if (!a || !b)
      {
-         return 1;
+         return true;
      }
 
+    assert(a->type == IP_ADDRESS_TYPE_IPV4 || a->type == IP_ADDRESS_TYPE_IPV6);
```

Comment:
Bad indentation.

Reply:
Most of the function is wrongly indented at 5 spaces, this is correct, 4 spaces.

Reply:
Fixed the rest of the indentation in a second commit.

Reply:
Refactored whole function and squashed into 1 commit.